user-service/src/main/java/com/ecommercepractice/userservice/service/UserService.java

```diff
@@ -0,0 +1,41 @@
+package com.ecommercepractice.userservice.service;
+
+
+import com.ecommercepractice.userservice.dao.IUserDao;
+import com.ecommercepractice.userservice.models.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class UserService {
```

Comment:
please use camel case format on this class

Reply:
Ok

Reply:
Carlos, its a class definition, so it should be UserService